### ISSUE TITLE
GameGuru MAX (alpha 2021-11-19 and probably latest upgrades) :  bug with the shadow of a transparent object.

### ISSUE THREAD

**Issue by marcuswilm** [TheGameCreators/GameGuruRepo#1672]
Test of a level rendered in GameGuru MAX alpha 2021-11-19 : the shadow of a transparent object is deactivated when the level is saved and reloaded. But the selection of the "Material" of that object in the "Object Tools" reactivates its shadow.

Screen capture showing the shadow correctly rendered for the transparent object : 
![480nm_02B](https://user-images.githubusercontent.com/94678034/145966571-f4c7c4a9-efaf-4168-bba8-dc14e15aed45.jpg)

And please see this very short video clip on Vimeo showing the process :  https://vimeo.com/656485034

As you know the material settings for the rendering on a object are stored in the .fpe file.
As far as I know, there are 12 settings available to define the material property of an object in GameGuru MAX.
But it seems that the activation of the shadow is outside of the scope of the .fpe file and that it could be an "intelligent" process in the game engine (am I right ?).

![fpe_visual_info](https://user-images.githubusercontent.com/94678034/145967465-2dc0c1b6-5401-4740-8614-bb98299eb327.jpg)

**Comment by plemsoft** [comment]
@marcuswilm Thanks for reporting this issue , yes it would be better to have it set while importing the model , anyway you can set it inside the .fpe , in your case mesh 2 need transparent , so you can add/change this in the .fpe:
"transparency2    = 1"

transparency2 , is for mesh 2 , transparency1 is for mesh 1 , and =1 is enable transparency , let me know if this fix your issue , thanks for your help :)

**Comment by marcuswilm** [comment]
@plemsoft :  Thank you for your e-mail. 

Unfortunately I am disappointed because I cannot obtain anymore that superb coloured shadow produced by the translucent wine glass when I test that scene with the alpha version 2022-02-18.
I consider that it is a major property of the "Materials" in GameGuru MAX because it is particularly interesting to render coloured shadows produced on a surface by the light going through **coloured stained glass**.

However I must admit that I don't understand your explanation relative to those terms : "transparency2 for mesh 2, transparency1 for mesh 1".  

What I wish is a correct combination of the parameters "**Transparent**" with "**Cast Shadows**" while the default value of the "**Alpha Channel**"  of the "**Base Color**" is modified to produce the desired level of transparency  (please note that I don't refer to the "Alpha Clipping" parameter). 

Using that combination of parameters worked nicely with GG MAX alpha 2021-11-19 in its editor **BUT** it was not saved correctly in the project. 

:-)
